apps/api/src/helpers/purl_helpers.ts

```diff
@@ -42,8 +42,3 @@ export const parsePurl = (purl: string) => {
         subpath: parsedPurl.subpath,
     };
 };
-
-// Replace %2F with / in, and /@ with /%40 in purl (in the purl spec, @ is used to separate the package name and the version)
-export const formatPurl = (purl: string) => {
-    return purl.replace(/%2F/g, "/").replace(/\/@/g, "/%40");
```

Comment:
Please briefly explain in the commit message *why* this replacement can be omitted now.